CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Log all available jmx metrics at debug level.
+* Misc logging improvements:
+    - Adds PID to startup message. This will help support customers who need to take a thread/heap dump for any reason.
+    - Clearer timezone identifier on logs (-07:00 instead of -07).
+    - add applicationinsights.testing.etw.disabled setting just for use during testing.
```

Comment:
i think we don't need to mention this? which pr has this change?

Reply:
https://github.com/microsoft/ApplicationInsights-Java/pull/1828

Reply:
we don't need to mention 3 changes in that pr.. just link line 5 with that pr.